pypeit/scripts/scriptbase.py

```diff
@@ -113,15 +117,14 @@ def entry_point(cls):
         """
         cls.main(cls.parse_args())
 
-    # TODO: Combining classmethod and property works in python 3.9 and later
-    # only: https://docs.python.org/3.9/library/functions.html#classmethod
-    # Order matters.  In python 3.9, it would be:
+    # TODO: This could also be combined with the property decorator; see
+    # https://docs.python.org/3.9/library/functions.html#classmethod
+    # Order matters.
     #
     # @classmethod
     # @property
     #
-    # Because we're not requiring python 3.9 yet, we have to leave this as a
-    # classmethod only:
+    # Leave as is for now.
```

Comment:
This was deprecated again in Python 3.11: https://docs.python.org/3.13/library/functions.html#classmethod

Reply:
Thanks for pointing that out!  I have removed the comment.